srcpkgs/udisks2/template

```diff
@@ -11,14 +11,14 @@ configure_args="--disable-static --with-udevdir=/usr/lib/udev
 hostmakedepends="docbook-xsl gettext glib-devel libxslt pkg-config
  $(vopt_if gir 'gobject-introspection') polkit"
 makedepends="acl-devel libatasmart-devel libgudev-devel libparted-devel
- polkit-devel sg3_utils-devel libblockdev-devel libbtrfs-devel elogind-devel"
+ polkit-devel sg3_utils-devel libblockdev-devel libbtrfs-devel"
```

Comment:
why?

Reply:
This removes features, there is no reason to remove it.

Reply:
I don't see any build options for it though..

Reply:
Re-added as a default build option